packages/x-license/src/useLicenseVerifier/useLicenseVerifier.test.tsx

```diff
@@ -105,5 +108,25 @@ describe('useLicenseVerifier', function test() {
       ]);
       expect(actualErrorMsg).to.match(/MUI X: Expired license key/);
     });
+
+    it('should throw if the license is not covering charts and tree-view', () => {
+      // Avoid Karma "Invalid left-hand side in assignment" SyntaxError
+      // eslint-disable-next-line no-useless-concat
+      process.env['NODE_' + 'ENV'] = 'development';
+
+      const initialLicenseKey = generateLicense({
+        expiryDate: new Date(3001, 0, 0, 0, 0, 0, 0),
+        orderNumber: 'MUI-123',
+        scope: 'pro',
+        licensingModel: 'subscription',
+        planVersion: 'initial',
+      });
+
+      LicenseInfo.setLicenseKey(initialLicenseKey);
+
+      expect(() => {
+        render(<TestComponent packageName={'x-charts-pro'} />);
```

Comment:
We also need to test:

- `x-tree-view-pro` with `initial` license key
- `x-charts-pro` and `x-tree-view-pro` with the `Q3-2024` key
- existing Pro and Premium packages with the `Q3-2924` key